files/lib/system/exporter/WBB4xExporter.class.php

```diff
@@ -2005,12 +2006,42 @@ public function countBlogs()
      */
     public function exportBlogs($offset, $limit)
     {
-        $sql = "SELECT      blog.*, language.languageCode
+        $sourceVersion52 = \version_compare(
+            $this->getPackageVersion('com.woltlab.blog'),
+            '5.2.0 Alpha 1',
+            '>='
+        );
+        $destVersion52 = \version_compare(
+            BLOGCore::getInstance()->getPackage()->packageVersion,
+            '5.2.0 Alpha 1',
+            '>='
+        );
```

Comment:
This seems obsolete to me, as the target version is always 6.0 or higher. `com.woltlab.wcf.exporter` 6.0 requires WoltLab Suite 6.0.